apps/react-storybook/stories/chat/Chat.stories.tsx

```diff
@@ -334,3 +334,84 @@ export const PopupIntegration: Story = {
         );
     }
 }
+
+export const Customization: Story = {
+    args: {
+        showDayHeaders: true,
+        showAvatar: true,
+        showUserName: true,
+        showMessageTimestamp: true,
+        items: initialMessages,
+        user: firstAuthor,
+        errors: [],
```

Comment:
I would prefer to simplify it as following:
```
export const Customization: Story = {
    args: {
        width: 500,
        height: 600,
        showDayHeaders: true,
        showAvatar: true,
        showUserName: true,
        showMessageTimestamp: true,
    },
    render: ({
        width,
        height,
        showDayHeaders,
        showAvatar,
        showUserName,
        showMessageTimestamp,
    }) => {
        return (
            <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
                <Chat
                    width={width}
                    height={height}
                    items={initialMessages}
                    user={secondAuthor}
                    showDayHeaders={showDayHeaders}
                    showAvatar={showAvatar}
                    showUserName={showUserName}
                    showMessageTimestamp={showMessageTimestamp}
                >
                </Chat>
            </div>
        );
    }
}
```

What do you think about this suggestion?

Reply:
Yep, it was a bit overcomplicated :)
b5b3568720d89710b638977cdb03e798e0aef78e